oidc-fapi-conformance-tests/export_results_fapi.py

```
@@ -17,35 +17,49 @@ def save_results(plan, i):
     test_fails = []
-    test_warnings = []
-    test_others = []
-    test_log = json.loads(requests.get(url=conformance_suite_url + "/api/log?length=100&search=" + plan['_id'], verify=False).content)
-    for test in test_log['data']:
-        if "result" in test and test['result'] == "FAILED":
-            test_fails.append('Test Name: ' + test['testName'] + '  id: ' + test['testId'])
-        elif "result" in test and test['result'] == "WARNING":
-            test_warnings.append('Test Name: ' + test['testName'] + '  id: ' + test['testId'])
+    test_errors = []
+    try:
+        response = requests.post(url=conformance_suite_url + "/api/plan/" + plan['_id'] + "/certificationpackage", verify=False)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)